packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -365,6 +375,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
         {label && (
           <Label id={id + '-label'} disabled={disabled} required={required} htmlFor={id + '-input'} className={this._classNames.label}>
             {label}
+            <div className={this._classNames.screenReaderText}>{multiselectPlaceholder}</div>
```

Comment:
>div [](start = 13, length = 3)

Consider using a `span`  instead of a `div` here

Reply:
And should this be `multiselectPlaceholder && <placeholder`

---
In reply to: [357327227](https://github.com/OfficeDev/office-ui-fabric-react/pull/11436#discussion_r357327227) [](ancestors = 357327227)

Reply:
If it's not visible does div vs span matter? I'll make a span either way.